File: 0217/solution.py
#https://leetcode.com/problems/contains-duplicate/

class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        hashtable = dict()
        for i in nums:
            if hashtable.get(i):
                return True
            else:
                hashtable[i] = 1
        return False

File: 0217/solution.java
//https://leetcode.com/problems/contains-duplicate/

class Solution {
    public boolean containsDuplicate(int[] nums) {
        Set<Integer> hashtable = new HashSet<Integer>();
        for (int i : nums) {
            if (!hashtable.add(i)) {
                return true;
            }
        }
        return false;
    }
}